File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        store = {}
        res = []
        for index, num in enumerate(nums):
            if num in store:
                res.append(store[num])
                res.append(index)
            else:
                store[target-num] = index
        return res

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int xorMiss = nums.length;
        for(int i = 0; i < nums.length; i++) {
            xorMiss = xorMiss ^ i ^ nums[i];
        }
        return xorMiss;
    }
}
